### PR TITLE
Refresh Apache Airflow configuration documentation

### LINKED ISSUE #1305: Refresh documentation to include Apache Airflow runtime environments
- Runtime configuration topic https://elyra.readthedocs.io/en/latest/user_guide/runtime-conf.html
- AI Pipelines https://elyra.readthedocs.io/en/latest/user_guide/pipelines.html
- https://elyra.readthedocs.io/en/latest/getting_started/overview.html#ai-pipelines-visual-editor
- https://elyra.readthedocs.io/en/latest/getting_started/overview.html#ability-to-run-a-notebook-as-a-batch-job

Tutorial is covered in https://github.com/elyra-ai/examples/issues/29

Address minor issues in https://elyra.readthedocs.io/en/latest/recipes/configure-airflow-as-a-runtime.html:
 - version information discrepancy
 - formatting
 - 404 link https://elyra.readthedocs.io/etc/kubernetes/airflow/helm/values.yaml
 - add link to topic in https://elyra.readthedocs.io/en/latest/recipes/configure-airflow-as-a-runtime.html#elyra-runtime-configuration-requirements

### PR DESCRIPTION
**Pull request by akchinSTC**
- Fixes for broken or missing links and formatting

Fixes #1305

Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 1]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
 - An Existing Apache Airflow Cluster 
-    - Ensure Apache Airflow is at least v1.10.8+, earlier versions may work (not tested)
+    - Ensure Apache Airflow is at least >=1.10.8,<2.0.0, earlier versions may work (not tested)
     - The [airflow-notebook](https://pypi.org/project/airflow-notebook/) python package installed on all schedulers/workers  
     - Using the Kubernetes Executor  
     
 ## Setting up a Github Repository
-A Github repository is needed to host your DAGs(pipelines). 
-
+A Github repository is needed to host your DAGs(pipelines).
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:46]:
```suggestion
A Github repository is needed to host your DAGs (pipelines).
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 2]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
+    - Ensure Kubernetes is >=1.18, earlier versions may work (not tested)
+    - Helm >=3.0
       OR
     - Use the [Helm chart](https://github.com/airflow-helm/charts/tree/main/charts/airflow) available in the Airflow source distribution with our [sample configuration](../../../etc/kubernetes/airflow/helm/values.yaml)  
     
 OR  
   
 - An Existing Apache Airflow Cluster 
-    - Ensure Apache Airflow is at least v1.10.8+, earlier versions may work (not tested)
+    - Ensure Apache Airflow is at least >=1.10.8,<2.0.0, earlier versions may work (not tested)
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:41]:
```suggestion
    - Ensure Apache Airflow is at least >= v1.10.8 and < v2.0.0. Other versions might work but have not been tested.
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 3]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
     
 ## Deploying Airflow on a new Kubernetes Cluster
-
+  
 ### SSH key requirements (Airflow Git-Sync)
 Apache Airflow uses a git-sync container to sync its collection of DAGs (pipelines) from a Github Repository. In order do so, 
 it requires an SSH key with read permissions to the repository added to the cluster as a kubernetes secret and provided during deployment.
-Please note that existing deployments may already have this step or equivalent already configured. 
-
+Please note that existing deployments may already have this step or equivalent already configured.
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:57]:
```suggestion
Please note that existing deployments may already have this step or equivalent already configured.
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 4]
```diff
@@ -99,30 +97,26 @@ For example, if you named your secret `airflow-secret` and github repository was
     ## if we should implicitly trust [git.repoHost]:git.repoPort, by auto creating a ~/.ssh/known_hosts
     ##
 ```
-
-4. Install Airflow with the [sample configuration](../../../etc/kubernetes/airflow/helm/values.yaml) or your own custom configuration 
-after making any other changes you might need to the configuration.
-
+  
+3. Install Airflow with the [sample configuration](https://raw.githubusercontent.com/elyra-ai/elyra/master/etc/kubernetes/airflow/helm/values.yaml) or your own custom configuration 
+after making any other changes you might need to the configuration.
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:102]:
```suggestion
after making any other changes you might need to the configuration.
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 5]
```diff
@@ -99,30 +97,26 @@ For example, if you named your secret `airflow-secret` and github repository was
...
+  
 ## Setting up Elyra on existing Airflow Deployments
-
-Getting Airflow to work with Elyra on existing deployments requires two things:
-
-- The deployment is synced to a Github Repository
-- The `airflow-notebook` python package is installed on the required Airflow pods (web-server, scheduler, workers)
-
+  
+Getting Airflow to work with Elyra on existing deployments requires two things:
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:113]:
```suggestion
Getting Airflow to work with Elyra on existing deployments requires two things:
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 6]
```diff
@@ -99,30 +97,26 @@ For example, if you named your secret `airflow-secret` and github repository was
...
-Getting Airflow to work with Elyra on existing deployments requires two things:
-
-- The deployment is synced to a Github Repository
-- The `airflow-notebook` python package is installed on the required Airflow pods (web-server, scheduler, workers)
-
+  
+Getting Airflow to work with Elyra on existing deployments requires two things:  
+  
+- The deployment is synced to a Github Repository  
+- The `airflow-notebook` python package is installed on the required Airflow pods (web-server, scheduler, workers)
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:116]:
```suggestion
- The `airflow-notebook` python package is installed on the required Airflow pods (web-server, scheduler, workers)
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 7]
```diff
@@ -99,30 +97,26 @@ For example, if you named your secret `airflow-secret` and github repository was
...
-
-Getting Airflow to work with Elyra on existing deployments requires two things:
-
-- The deployment is synced to a Github Repository
-- The `airflow-notebook` python package is installed on the required Airflow pods (web-server, scheduler, workers)
-
+  
+Getting Airflow to work with Elyra on existing deployments requires two things:  
+  
+- The deployment is synced to a Github Repository
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:115]:
```suggestion
- The deployment is synced to a Github Repository
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md (hunk 2 above)
Comment (akchinSTC) [reply docs/source/recipes/configure-airflow-as-a-runtime.md]: done

File: docs/source/recipes/configure-airflow-as-a-runtime.md (hunk 1 above)
Comment (akchinSTC) [reply docs/source/recipes/configure-airflow-as-a-runtime.md]: done

**Comment by akchinSTC** [comment]
Rebased and updated based on comments @ptitzler

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 8]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
-
-This guide assumes a general working knowledge of kubernetes and administration of a cluster
-
+the Airflow Deployment on Kubernetes and Elyra's runtime configuration.  
+  
+This guide assumes a general working knowledge of and administration of a Kubernetes cluster
 ## Requirements
-
-- A Private Github Repository
+
```
Comment (lresende) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:27]: there are a few white spaces issues ...

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 9]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
+    - Ensure Kubernetes is >=1.18, earlier versions may work (not tested)
+    - Helm >=3.0
       OR
     - Use the [Helm chart](https://github.com/airflow-helm/charts/tree/main/charts/airflow) available in the Airflow source distribution with our [sample configuration](../../../etc/kubernetes/airflow/helm/values.yaml)  
     
 OR  
   
 - An Existing Apache Airflow Cluster 
-    - Ensure Apache Airflow is at least v1.10.8+, earlier versions may work (not tested)
+    - Ensure Apache Airflow is at least >= v 1.10.8 and < v2.0.0. Other versions might work but have not been tested.
```
Comment (lresende) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:41]:
```suggestion
    - Ensure Apache Airflow is at least v1.10.8 and below v2.0.0. Other versions might work but have not been tested.
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 10]
```diff
@@ -99,30 +97,26 @@ For example, if you named your secret `airflow-secret` and github repository was
     ## if we should implicitly trust [git.repoHost]:git.repoPort, by auto creating a ~/.ssh/known_hosts
     ##
 ```
-
-4. Install Airflow with the [sample configuration](../../../etc/kubernetes/airflow/helm/values.yaml) or your own custom configuration 
+  
+3. Install Airflow with the [sample configuration](https://raw.githubusercontent.com/elyra-ai/elyra/master/etc/kubernetes/airflow/helm/values.yaml) or your own custom configuration
```
Comment (lresende) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:101]: Do we need to update the release script to change /master/ to /vx.y.z/?

**Comment by ptitzler** [comment]
`make docs` triggers warninga:
```
/Users/patti/elyra_test/airflow-docs/elyra/docs/source/index.rst:60: WARNING: toctree contains reference to nonexisting document 'developer_guide/deploying-kubeflow-locally-for-dev'

/Users/patti/elyra_test/airflow-docs/elyra/docs/source/developer_guide/release.md: WARNING: document isn't included in any toctree
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 11]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
-This guide assumes a general working knowledge of kubernetes and administration of a cluster
-
+the Airflow Deployment on Kubernetes and Elyra's runtime configuration.  
+  
+This guide assumes a general working knowledge of and administration of a Kubernetes cluster
 ## Requirements
-
-- A Private Github Repository
+  
+- A Private Github Repository
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:28]: A private repository on github.com or Github Enterprise
```suggestion
- A private repository on github.com or Github Enterprise.
```

**Comment by ptitzler** [comment]
Looks like the official names are `GitHub` and `GitHub Enterprise` (note the case)

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 12]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
 # Configuring Airflow on Kubernetes for use with Elyra (Experimental)
 
 Using Apache Airflow with Elyra requires configuring two parts,
-the Airflow Deployment on Kubernetes and Elyra's runtime configuration.
-
-This guide assumes a general working knowledge of kubernetes and administration of a cluster
-
+the Airflow Deployment on Kubernetes and Elyra's runtime configuration.
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:23]:
```suggestion
the Airflow deployment on Kubernetes and Elyra's runtime configuration.  
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 13]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
 - An Existing Apache Airflow Cluster 
-    - Ensure Apache Airflow is at least v1.10.8+, earlier versions may work (not tested)
+    - Ensure Apache Airflow is at least >= v 1.10.8 and < v2.0.0. Other versions might work but have not been tested.
     - The [airflow-notebook](https://pypi.org/project/airflow-notebook/) python package installed on all schedulers/workers  
     - Using the Kubernetes Executor  
     
 ## Setting up a Github Repository
-A Github repository is needed to host your DAGs(pipelines). 
-
+A Github repository is needed to host your DAGs (pipelines).
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:46]:
```suggestion
A GitHub repository is needed to host your DAGs (pipelines).
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 14]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
+     
 ```bash
 kubectl create secret generic airflow-secret --from-file=id_rsa=.ssh/id_rsa --from-file=known_hosts=.ssh/known_hosts --from-file=id_rsa.pub=.ssh/id_rsa.pub -n airflow
-```
-
+```  
+  
 2. After creating your secret, enter the name of the kubernetes secret in the sample configuration for `helm` or in your own custom configuration.
-Also, update the `git url` section to the private github repository created [earlier](#setting-up-a-github-repository)  
+Also, update the `git url` section to the private github repository created [earlier](#setting-up-a-github-repository).
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:67]:
```suggestion
Also, update the `git url` section to the private GitHub repository created [earlier](#setting-up-a-github-repository).  
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 15]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
+  
 - A Kubernetes Cluster 
-    - Ensure Kubernetes is at least v1.18+, earlier versions may work (not tested)
-    - Helm v3.0  
-    - Great guide on how to deploy an Airflow cluster:
-    [Apache Airflow with Kubernetes Executor and Minikube](https://marclamberti.com/blog/airflow-kubernetes-executor/)
+    - Ensure Kubernetes is >=1.18, earlier versions may work (not tested)
+    - Helm >=3.0
       OR
     - Use the [Helm chart](https://github.com/airflow-helm/charts/tree/main/charts/airflow) available in the Airflow source distribution with our [sample configuration](../../../etc/kubernetes/airflow/helm/values.yaml)
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:36]:
```suggestion
    - Use the [Helm chart](https://github.com/airflow-helm/charts/tree/main/charts/airflow) available in the Airflow source distribution with our [sample configuration](https://raw.githubusercontent.com/elyra-ai/elyra/master/etc/kubernetes/airflow/helm/values.yaml)  
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 16]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
+  
 - You will need to setup two things:
     - [Generate an SSH key to be used as a secret for Airflow](https://docs.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account)
     - [A Github Access Token with write access to the repo](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
     
 ## Deploying Airflow on a new Kubernetes Cluster
-
+  
 ### SSH key requirements (Airflow Git-Sync)
 Apache Airflow uses a git-sync container to sync its collection of DAGs (pipelines) from a Github Repository. In order do so,
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:55]:
```suggestion
Apache Airflow uses a git-sync container to sync its collection of DAGs (pipelines) from a GitHub Repository. In order do so, 
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 17]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
 - You will need to setup two things:
     - [Generate an SSH key to be used as a secret for Airflow](https://docs.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account)
     - [A Github Access Token with write access to the repo](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
     
 ## Deploying Airflow on a new Kubernetes Cluster
-
+  
 ### SSH key requirements (Airflow Git-Sync)
 Apache Airflow uses a git-sync container to sync its collection of DAGs (pipelines) from a Github Repository. In order do so, 
 it requires an SSH key with read permissions to the repository added to the cluster as a kubernetes secret and provided during deployment.
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:56]:
```suggestion
it requires an SSH key with read permissions to the repository added to the cluster as a Kubernetes secret and provided during deployment.
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md (hunk 14 above)
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:67]:
```suggestion
Also, update the `git url` section to the private GitHub repository created [earlier](#setting-up-a-github-repository).  
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 18]
```diff
@@ -20,55 +20,53 @@ limitations under the License.
...
 ```bash
 kubectl create secret generic airflow-secret --from-file=id_rsa=.ssh/id_rsa --from-file=known_hosts=.ssh/known_hosts --from-file=id_rsa.pub=.ssh/id_rsa.pub -n airflow
-```
-
+```  
+  
 2. After creating your secret, enter the name of the kubernetes secret in the sample configuration for `helm` or in your own custom configuration.
-Also, update the `git url` section to the private github repository created [earlier](#setting-up-a-github-repository)  
+Also, update the `git url` section to the private github repository created [earlier](#setting-up-a-github-repository).  
 For example, if you named your secret `airflow-secret` and github repository was under the `elyra` organization and named `examples` under the
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:68]:
```suggestion
For example, if you named your secret `airflow-secret` and GitHub repository was under the `elyra` organization and named `examples` under the 
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 19]
```diff
@@ -99,30 +97,26 @@ For example, if you named your secret `airflow-secret` and github repository was
...
   
-
-
 ```bash
-helm install "airflow" stable/airflow --values https://raw.github.com/elyra-ai/elyra/etc/kubernetes/helm/airflow/values.yaml
+helm install "airflow" stable/airflow --values https://raw.githubusercontent.com/elyra-ai/elyra/master/etc/kubernetes/airflow/helm/values.yaml
 ```
-
+  
 ## Setting up Elyra on existing Airflow Deployments
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:111]:
```suggestion
## Setting up Elyra on existing Airflow deployments
```

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 20]
```diff
@@ -99,30 +97,26 @@ For example, if you named your secret `airflow-secret` and github repository was
...
+  
 ## Elyra Runtime Configuration Requirements
-
-1. A Github Personal Access Token under the user/organization where the DAG repository is located. This step was done [earlier](#setting-up-a-github-repository).
-
-2. Add the token to your Airflow metadata runtime configuration 
-
-
+  
+1. A Github Personal Access Token under the user/organization where the DAG repository is located. This step was done [earlier](#setting-up-a-github-repository).
```
Comment (ptitzler) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:120]:
```suggestion
1. A GitHub Personal Access Token under the user/organization where the DAG repository is located. This step was done [earlier](#setting-up-a-github-repository).  
```

**Review by lresende — APPROVED** [review]

**Pull request merged**